WebNetSample/Views/Shared/_HomeLayout.cshtml

```diff
@@ -0,0 +1,76 @@
+    <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" />
+    <link rel="stylesheet" href="~/WebNetMVC.styles.css" asp-append-version="true" />
+</head>
+<body>
+    <header>
+            <nav class="navbar navbar-expand-sm navbar-toggleable-sm navbar-light bg-white border-bottom box-shadow mb-3">
+                <div class="container-fluid">
+                    <a class="navbar-brand" asp-area="" asp-controller="Home" asp-action="Index">WebNetSample</a>
+                    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target=".navbar-collapse" aria-controls="navbarSupportedContent"
+                            aria-expanded="false" aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon"></span>
+                    </button>
+                    <div class="navbar-collapse collapse d-sm-inline-flex justify-content-between">
+                        <ul class="navbar-nav flex-grow-1">
+                            <li class="nav-item">
+                                <a class="nav-link text-dark" asp-controller="Home" asp-action="Index">Home</a>
+                            </li>
+                            <li class="nav-item dropdown">
+                                <button class="btn dropdown-toggle" type="button" id="dropdownMenuButton1" data-bs-toggle="dropdown" aria-expanded="false">
+                                    Categories
+                                </button>
+                                <ul class="dropdown-menu" aria-labelledby="dropdownMenuButton1">
+                                    <li><a class="dropdown-item" asp-controller="Categories" asp-action="Index">Toy</a></li>
+                                    <li><a class="dropdown-item" asp-controller="Categories" asp-action="Index">Food</a></li>
+                                    <li><div class="dropdown-divider"></div></li>
+                                    <li><a class="dropdown-item" asp-controller="Categories" asp-action="Index">All</a></li>
+                                </ul>
+                                <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+
+                                </div>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link text-dark" asp-controller="Products" asp-action="Index">Products</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link text-dark" asp-controller="Suppliers" asp-action="Index">Suppliers</a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </nav>
+    </header>
+
+    <div class="container">
+        <main role="main" class="pb-3">
+            @if (IsSectionDefined("Images"))
+            {
+                await RenderSectionAsync("Images", false);
```

Comment:
Just a FYI: in **strongly** advice you to use named params in C# or put all params into variables and assign them as function arguments. Some people may not be too familiar with code/technology and `false` value as second argument does not give them any meaning, they will have to spend time on investigations. Try to always use name params approach, like:
```suggestion
                await RenderSectionAsync("Images", required: false);
```